tsl/test/sql/compression_ddl.sql

```diff
@@ -489,8 +497,10 @@ FROM generate_series('2018-03-02 1:00'::TIMESTAMPTZ, '2018-03-02 13:00', '1 hour
 
 ALTER TABLE test2 set (timescaledb.compress, timescaledb.compress_segmentby = 'i', timescaledb.compress_orderby = 'timec');
 
-SELECT relname FROM pg_class
-WHERE reltablespace in
+-- the toast table and its index will have a different name depending on
+-- the order the tests run, so anonymize it
+SELECT regexp_replace(relname, 'pg_toast_[0-9]+', 'pg_toast_XXX') FROM pg_class where relname like '%pg_toast%'
```

Comment:
What is the reason behind the restriction `relname like '%pg_toast%'`? It would be good to check if the chunks are actually created in this tablespace.

Reply:
Thank you for pointing this out, indeed I did not mean to add the restriction but I wasn't careful here... I have removed it now.